distributed/tests/test_ipython.py

```diff
@@ -15,11 +14,7 @@ def need_functional_ipython(func):
         import jupyter_client  # noqa: F401
     except ImportError:
         return pytest.mark.skip("need ipykernel and jupyter_client installed")(func)
-    if tornado.version_info >= (5,):
```

Comment:
tornado 5 is required by both ipykernel and distributed, so this condition is always True.

The upstream bug has been fixed for some time.

Reply:
Yeah we also [require Tornado 5+ as well (if not newer)]( https://github.com/dask/distributed/blob/4ce8c65039e9b531878f9d77cf884920690a3745/requirements.txt#L9-L10 ). So this was probably just missed when we bumped the requirement